cclient/serialization/json/json_serializer.c

```diff
@@ -304,13 +321,72 @@ retcode_t json_get_balances_deserialize_response(const serializer_t* const s,
 }
 
 // get_inclusion_state_response
-void json_get_inclusion_state_serialize_request(
-    const serializer_t* const s, const get_inclusion_state_req_t* const obj,
-    char* out) {}
+retcode_t json_get_inclusion_state_serialize_request(
+    const serializer_t* const s, get_inclusion_state_req_t* obj,
+    char_buffer_t* out) {
+  retcode_t ret = RC_OK;
+  const char* json_text = NULL;
+  size_t len = 0;
+  cJSON* json_root = cJSON_CreateObject();
+  if (json_root == NULL) {
+    return RC_CCLIENT_JSON_CREATE;
+  }
+
+  cJSON_AddItemToObject(json_root, "command",
+                        cJSON_CreateString("getInclusionStates"));
+
+  ret = utarray_to_json_array(obj->hashes, json_root, "transactions");
+  if (ret != RC_OK) {
+    goto err;
+  }
 
-void json_get_inclusion_state_deserialize_response(
+  ret = utarray_to_json_array(obj->tips, json_root, "tips");
+  if (ret != RC_OK) {
+    goto err;
+  }
+
+  json_text = cJSON_PrintUnformatted(json_root);
+  if (json_text) {
+    len = strlen(json_text);
+    ret = char_buffer_allocate(out, len);
+    if (ret == RC_OK) {
+      strncpy(out->data, json_text, len);
+    }
+    cJSON_free((void*)json_text);
+  }
+
+  cJSON_Delete(json_root);
+  return ret;
+
+err:
+  cJSON_Delete(json_root);
+  return ret;
```

Comment:
Little code duplication, can we call the label `done` and remove one of the duplicates ?